python/graphstorm/graphpartitioning/array_readwriter/parquet.py

```diff
@@ -0,0 +1,82 @@
+import logging
+
+import numpy as np
+import pandas as pd
+import pyarrow
+import pyarrow.parquet
+
+from .registry import register_array_parser
+
+
+@register_array_parser("parquet")
+class ParquetArrayParser(object):
```

Comment:
it seems that you are defining your own format to read data. Who prepared the data?

Reply:
This is just a class decorator, for registering this class for read/write operations on parquet files. 

This class is borrowed from the existing pipeline's implementation and the original author of these set of classes is Minjie. That said, these set of class expect data to be read and returned to the invoker in numpy format. Similarly the numpy data is taken and written in the appropriate files, as defined by the user.